testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -96,6 +96,12 @@
 
     SUPPORTS_MULTI_STATEMENT_WRITES(false),
 
+    SUPPORTS_TIMESERIES_COLUMN(false),
+
+    SUPPORTS_SCALAR_QUERIES(true),
+
+    SUPPORTS_HAVING_CLAUSE(true),
+
```

Comment:
Please revert these change.

Reply:
As mentioned with another comment, these were introduced because Pinot wasn't passing some of the BaseConnector tests.

Reply:
Please see other comment https://github.com/trinodb/trino/pull/13619#discussion_r950909571